src/main/java/lotto/domain/LottoStatistics.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static lotto.domain.LottoInformation.*;
+
+public class LottoStatistics {
+
+    private final Map<Integer,  Integer> statisticsMap;
```

Comment:
몇개가 당첨되었는지 기록하는 클래스같아요
```suggestion
    private final Map<LottoInformation,  Integer> statisticsMap;
```

LottoInformation 을 키로 가지고가도 되지 않을까요?